scripts/install_mt4.verb

```diff
@@ -22,12 +22,10 @@ load_install_mt4()
     w_try_cd "$W_CACHE/$W_PACKAGE"
     w_ahk_do "
         Run, "$(find . -name "*.exe" -print -quit)"
-        WinWait, 4 Setup
-        ControlClick, Button1
-        Sleep 100
-        ControlClick, Button3
-        WinWait, 4 Setup, Installation successfully completed
-        ControlClick, Button4
+        WinWait, XM UK MT4 Setup
+        Send, {Enter}
+        WinWait, XM UK MT4 Setup, Congratulations! We are waiting for you...
```

Comment:
As above, it's possible to make it more universal? Otherwise it'll fail on a single character change.

Reply:
Probably, but I'll need to change how AHK matches window titles. By default, it only matches from the beginning of the title.

Reply:
I think we can do [`SetTitleMatchMode, 2`](https://www.autohotkey.com/docs/commands/SetTitleMatchMode.htm).

Reply:
Yes, that's what I'm testing now.

Reply:
And the modification is complete. Is there anything else?